packages/frontend/editor-ui/src/components/TagsDropdown.vue

```diff
@@ -59,7 +59,9 @@ const container = ref<HTMLDivElement>();
 const dropdownId = uuid();
 
 const options = computed<ITag[]>(() => {
-	return props.allTags.filter((tag: ITag) => tag && tag.name.includes(filter.value));
+	return props.allTags.filter(
+		(tag: ITag) => tag && tag.name.toLowerCase().includes(filter.value.toLowerCase()),
```

Comment:
Nitpick: I think tag existence is guaranteed at this point so we can simplify this.
```suggestion
		(tag: ITag) => tag.name.toLowerCase().includes(filter.value.toLowerCase()),
```